File: Java/day3/Atm.java
import java.util.Scanner;

public class Atm {
    public static void main(String[] args) {
        /*
         * Withdraw Money
         * -> amt < bal
         * -> 500 notes available
         */

         int amt, bal = 10000;
         Scanner sc = new Scanner(System.in);

         System.out.print("Enter amount to withdraw: ");
         amt = sc.nextInt();

         if(amt < bal){
            if(amt%500==0){
                System.out.println(amt + " amount withdrawed.");
                bal -= amt;
                System.out.println(bal + " balance remaining.");
            }
            else{
                System.out.println("Only 500 notes are available.");
            }
         }
         else{
            System.out.println("Insufficient balance.\n" + bal + " balance available.");
         }

    }
}

File: CLang/day8/atm.c
/*
    -> Withdraw Money
        amt < bal
        atm has 500 notes only

    -> Deposit Money
        amt >= 5000
        atm only accept 500 notes

    -> Show Balance

    -> Exit

*/

#include<stdio.h>

void main(void){
    int amt, bal = 10000, opt;
    int key = 1;
    printf("1. Withdraw Money\n");
    printf("2. Deposit Money\n");
    printf("3. Show Balance\n");
    printf("4. Exit\n");

    while(key == 1){
        printf("\nSelect Option: ");
        scanf("%d",&opt);

        if(opt == 1){
            printf("\nEnter amount to withdraw: ");
            scanf("%d",&amt);
            if(amt < bal){
                if(amt%500==0){
                    printf("%d amount withdrawed.\n",amt);
                    bal = bal - amt;
                }
                else{
                    printf("Only 500 notes are available");
                }
            }
            else{
                printf("Insufficient Balance\n");
                printf("Available balance: %d\n",bal);
            }
        }
        else if(opt == 2){
            printf("Deposit Money");
        }
        else if(opt == 3){
            printf("Available balance: %d\n",bal);
        }
        else if(opt == 4){
            printf("Visit Again :)");
            key = 0;
            break;
        }
    }

}